packages/esm-patient-orders-app/src/lab-results/lab-results-form.scss

```diff
@@ -0,0 +1,78 @@
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
+@import '~@openmrs/esm-styleguide/src/vars';
+
+section {
```

Comment:
This style rule will be applied globally. We should remove it and instead apply a value like 3 to the `gap` prop of the `Stack` component in `LabResultsForm`.

Reply:
Renamed